lms/envs/aws.py

```diff
@@ -191,6 +191,13 @@
 CONTACT_EMAIL = ENV_TOKENS.get('CONTACT_EMAIL', CONTACT_EMAIL)
 BUGS_EMAIL = ENV_TOKENS.get('BUGS_EMAIL', BUGS_EMAIL)
 PAYMENT_SUPPORT_EMAIL = ENV_TOKENS.get('PAYMENT_SUPPORT_EMAIL', PAYMENT_SUPPORT_EMAIL)
+PAYMENT_CONFIRM_EMAIL = ENV_TOKENS.get(
+    'PAYMENT_CONFIRM_EMAIL',
+    ENV_TOKENS.get(
+        'PAYMENT_SUPPORT_EMAIL',
+        PAYMENT_CONFIRM_EMAIL
```

Comment:
Yeah, that's the gist. It's a bit more clear starting from `lms/envs/common.py`, where you can see that we want `PAYMENT_CONFIRM_EMAIL` to default to `PAYMENT_SUPPORT_EMAIL`. Then to accomplish the same thing here, we'll
- try to use the value of `*_CONFIRM_*` from JSON
- try to use the value of `*_SUPPORT_*` from JSON
- fall back to the value of `*_CONFIRM_*`
- default to value of `*_SUPPORT_*` (implicitly from `lms/envs/common.py`)

Does that make sense?